consent-ws/src/test/java/org/broadinstitute/consent/http/ConsentManageTest.java

```diff
@@ -49,15 +49,30 @@ public void testConsentManage() throws IOException {
                 .equals(ElectionStatus.OPEN.getValue()));
         Assert.assertTrue(consentManageUpdated.get(0).getElectionStatus()
                 .equals(ElectionStatus.OPEN.getValue()));
+
         List<Vote> votes = getJson(client, voteConsentPath(CONSENT_ID)).readEntity(new GenericType<List<Vote>>() { });
         deleteVotes(votes, CONSENT_ID);
         delete(client, electionConsentPathById(CONSENT_ID, electionId));
         List<Vote> votesElection2 = getJson(client, voteConsentPath(CONSENT_ID_2)).readEntity(new GenericType<List<Vote>>() {
         });
+        // find election by vote id
+        testFindElectionByVoteId(votesElection2.get(0));
         deleteVotes(votesElection2, CONSENT_ID_2);
         delete(client, electionConsentPathById(CONSENT_ID_2, electionId_2));
     }
 
+    @Test
+    public void testFindElectionByNonExistentVoteId()  throws IOException {
+        Client client = ClientBuilder.newClient();
+        checkStatus(NOT_FOUND, getJson(client, electionVotePathById(999)));
+    }
+
+    private void testFindElectionByVoteId(Vote vote) throws IOException {
```

Comment:
one nit: it actually takes a vote object and not an Id object. maybe name the method testFindElectionByVote

Reply:
:+1: